inventory/host_vars/ocp11.yaml

```diff
@@ -42,3 +46,16 @@ cluster_capabilities:
 #  - Ingress
 #  - OperatorLifecycleManager
 #  - marketplace
+
+cluster_additionalTrustBundle: ./Current-IT-Root-CAs.crt
```

Comment:
@DanielFroehlich where is the file located?

Reply:
Its in the home directory of the exec env. I think I got this from here:
https://github.com/stormshift/automation/blob/db06a1a87d6dad1b037aa6bf87ec2e5a620e8959/README.md?plain=1#L110

Reply:
I'm a bit surprised because it's part of the build process: https://github.com/stormshift/automation/blob/master/execution-environment.yml

My solutions are not better:
1) Fetch root vault: https://github.com/stormshift/automation/blob/hcp-deployment/roles/cluster-hcp/tasks/deploy-cluster.yaml#L211 
2) The bad one hard coded: https://github.com/stormshift/automation/blob/hcp-deployment/roles/cluster/tasks/post-configuration.yaml#L6

Reply:
Or we fetch from: https://certs.corp.redhat.com/certs/Current-IT-Root-CAs.pem ?

Reply:
I mean, its a public CA, so I guess it would be okay'ish to simply put it to githup. But as its internal, probably better not. 
I like the "download during the execution env build" - then it is "simply there"

Another option: How about adding it to `development-example.vars-private`, next to the pull secret?

Reply:
https://github.com/stormshift/automation/issues/98 - let's add the Root CA into the execution env. As file.